docs/static/ls-ls-http.asciidoc

```diff
@@ -8,3 +8,126 @@ NOTE: Check out these <<http-considerations,considerations>> before you implemen
 
 For now, <<plugins-outputs-http,http output>> to <<plugins-inputs-http,http input>> with manual configuration may be the best path forward if these limitations don't apply to your use case.
 
+==== Configuration overview
+
+To use the HTTP protocol to connect two Logstash instance you will:
+
+. Configure the downstream (server) Logstash to use HTTP input
+. Configure the upstream (client) Logstash to use HTTP ouput
+. Securing the communication between HTTP input and HTTP output
+
+[[configure-downstream-logstash-http-input]]
```

Comment:
You can shorten this if you want to, but you don't have to. The only requirement is that the anchor be unique in context of this guide.